isis/src/mex/apps/hrsc2isis/main.cpp

```diff
@@ -172,30 +234,24 @@ void IsisMain() {
   // Get as many of the other labels as we can
   Pvl otherLabels;
 
-  //p.TranslatePdsLabels (otherLabels);
   TranslateHrscLabels(label, otherLabels);
 
-  if(otherLabels.hasGroup("Mapping") &&
```

Comment:
I think I understand why this was removed, given that mapping explicitly isn't supported. Was it supported before?

Reply:
Resolved: it was never supported.

Reply:
No it was not. From what I can tell this code was copied directly from pds2isis. The mapping section was just not deleted.